File: Java/0001_two_sum.java
class MySolution {
    public int[] twoSum(int[] nums, int target) {
        int[] res = new int[2];
        boolean hasFound = false;
        for (int i = 0; i < nums.length; i++)
        {
            int tmp = target - nums[i];
            for (int j = i + 1; j < nums.length; j++)
            {
                if (nums[j] == tmp)
                {
                    res[0] = i;
                    res[1] = j;
                    hasFound = true;
                    break;
                }
            }
            if (hasFound == true)
                break;
        }
        return res;
    }
}

File: C++/0001_two_sum.cpp
#include <iostream>
#include <vector>

using std::vector;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> indices;
        for (int i = 0; i < nums.size() - 1; i++)
        {
            for (int j = i + 1; j < nums.size(); j++)
            {
                if (nums[i] + nums[j] == target)
                {
                    indices.push_back(i);
                    indices.push_back(j);
                    break;
                }
            }
        }
        return indices;
    }
};
